docs/prow/security_commit_scanner.md

```diff
@@ -0,0 +1,39 @@
+# Security Leaks Scanner
+
+Welcome to the Security Leaks Scanner, a tool designed to scan repository for potential 
```

Comment:
In technical documentation, the style is simple so that the info is easy to understand and follow even for non-native users of English. For more information, you can see the [content guidelines](https://github.com/kyma-project/community/tree/main/docs/guidelines/content-guidelines) here.